### PR TITLE
Dockerfile addition - Run with the Dockers, Build with the Dockers

### LINKED ISSUE #31: Doc update needed
Thanks for creating an issue! Please fill out this form so we can be
sure to have all the information we need, and to minimize back and forth.

* **What are you trying to do?**
Install

* **What feature or behavior is this required for?**
Documentation

* **How could we solve this issue? (Not knowing is okay!)**
The (most) Permanent Install steps need to be changed. After following those steps the plugin is not loaded and an R repository cannot be created.

* **Anything else?**
From internal JIRA ticket https://issues.sonatype.org/browse/NEXUS-13418

### PR DESCRIPTION
**Pull request by DarthHater**
This Dockerfile addition is a no shame imitation of the cool work @mpoindexter did in apt, and I'm trying to spread to our other community projects.

This pull request makes the following changes:
* Adds Dockerfile, builds R and then adds it to Nexus Repo
* Adds documentation on this

Might fix #31 , jury is out

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+ARG NEXUS_VERSION=3.10.0
```
Comment (jlstephens89) [inline Dockerfile]: Might want to update to 3.13 now 😄

**Comment by pachiras** [comment]
Nice work!
It worked perfectly :)

**Comment by bhamail** [comment]
I think we should head in this direction with simpler [Dockerfile example](https://github.com/sonatype-nexus-community/nexus-format-archetype/blob/master/src/test/resources/projects/it1/reference/Dockerfile).

**Comment by aornatovskyy** [comment]
@bhamail @DarthHater  could this ticket be merged and closed?

**Comment by bhamail** [comment]
@aornatovskyy I pushed a WIP branch that contains a "simplified" Dockerfile. However, that branch depends on a new NXRM (with .kar support). I'm closing this PR, so long as you create a new Issue and/or PR #108 gets addressed later.

**Pull request closed without merging**